src/containers/documentation/ApiDocumentation.tsx

```diff
@@ -1,40 +1,61 @@
 import * as React from 'react';
+import { connect } from 'react-redux';
+import { Dispatch } from 'redux';
+import * as actions from '../../actions';
 
 import { Flag } from 'flag';
 import { Location } from 'history';
 import { Tab, TabList, TabPanel, Tabs } from 'react-tabs';
+import SwaggerDocs from './SwaggerDocs';
 
 import { lookupApiCategory } from '../../apiDefs/query';
 import { IApiDescription, IApiDocSource } from '../../apiDefs/schema';
 import { history } from '../../store';
-import SwaggerDocs from './SwaggerDocs';
+import { IRootState } from '../../types';
+
 
 import '../../../node_modules/react-tabs/style/react-tabs.scss';
 
 interface IApiDocumentationProps {
   apiDefinition: IApiDescription;
   categoryKey: string;
   location: Location;
+  requestedApiVersion: string;
```

Comment:
```suggestion
  apiVersion: string;
```

see comment in `src/types/index.ts`